sklearn/linear_model/base.py

```diff
@@ -420,7 +492,7 @@ def fit(self, X, y, sample_weight=None):
         y : numpy array of shape [n_samples, n_targets]
             Target values
 
-        sample_weight : numpy array of shape [n_samples]
+        sample_weight : float or numpy array of shape [n_samples]
```

Comment:
I think some tests used already float weights before. I have just updated the docstring.